Resources/Prototypes/Nyanotrasen/Entities/Structures/Research/glimmer_prober.yml

```diff
@@ -65,6 +65,25 @@
   - type: GuideHelp
     guides:
     - Psionics
+  - type: AmbientSound
+    range: 6
+    volume: -6
+    sound: /Audio/Ambience/Objects/prober_hum_low.ogg
+  - type: AmbientOnPowered
+  - type: GlimmerSound
+    glimmerTier:
+      Minimal:
+        path: /Audio/Ambience/Objects/prober_hum_low.ogg
+      Low:
+        path: /Audio/Ambience/Objects/prober_hum_low.ogg
+      Moderate:
+        path: /Audio/Ambience/Objects/prober_hum_moderate.ogg
+      High:
+        path: /Audio/Ambience/Objects/prober_hum_high.ogg
+      Dangerous:
```

Comment:
The change to above 500 needs to be a different sound since that's the most important threshold for a bunch of reasons. So we can either change the High sound back to prober_hum_moderate.ogg, or make a fourth sound

Reply:
Good point, added a higher stage.